src/student.rb

```diff
@@ -9,4 +9,6 @@ def initialize(age, classroom, name = 'unknown', parent_permission: true)
   def play_hooky
     '¯\(ツ)/¯'
   end
+
+  belongs_to :classroom
```

Comment:
- [ ] The has-many / belongs-to relationships between Classroom and Student should be implemented according to this [article](`https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships`). Kindly take a careful look at the requirements.